cmd/linux.go

```diff
@@ -235,6 +236,10 @@ func linuxCmd(_ *shell.Interface, arg []string) (res string, err error) {
 		return "", fmt.Errorf("could not open root volume, %v", err)
 	}
 
+	if err = transparency.Check(root, false); err != nil {
```

Comment:
See earlier comment, this feels it should be conditional to configuration/command arguments and not a build tag.

It is also inadvisable to have a sort of NOP opaque default as someone might miss it by just reading this source file.